frontend/src/components/APRRateDisplay.js

```diff
@@ -522,21 +568,15 @@ class APRRateDisplay extends Component {
           }}
           disable={true}
         />
-        {/*<div class="col-lg-4 pull-right">
-          <ul class="nav nav-pills nav-stacked">
-            <button onClick={() => this.createChart('bar')}>Bar Chart</button>
-            <button onClick={() => this.createChart('pie')}>Pie Chart</button>
-            <button onClick={() => this.createChart('line')}>Line Chart</button>
-            <button onClick={() => this.createChart('tree')}>TreeMap</button>
-            <li role="presentation" class="active">
-              <a href="#">Line Chart</a>
-            </li>
-            <li role="presentation">
-              <a href="#">Pie Chart</a>
-            </li>
-          </ul>
-        </div>
-        <KivaChart visualType={this.state.visualType} />*/}
+        <div class = "col-lg-4 pull-right">
+		<ul class = "nav nav-pills nav-stacked">
+			<li role = "presentation" class={this.state.barclass}><a onClick={()=>this.changeChart("bar")}>Bar</a></li>
+			<li role = "presentation" class={this.state.lineclass}><a onClick={()=>this.changeChart("line")}>Line</a></li>
+			<li role = "presentation" class={this.state.pieclass}><a onClick={()=>this.changeChart("pie")}>Pie</a></li>
+			<li role = "presentation" class={this.state.areaclass}><a onClick={()=>this.changeChart("area")}>Area</a></li>
+		</ul>
```

Comment:
use 2 space indenting as well as make the kivachart component in line with the other components.